subcommittees/README.md

```diff
@@ -21,6 +21,7 @@ task).
 * [Finance](finance/README.md)
 * [Lesson Organization and Develpment](lessons/README.md)
 * [Partner Relations and Fundraising](partner-relations/README.md)
+* [Code of Conduct and Policy](policy-CoC/README.md)
```

Comment:
The name of this subcommittee is "Policy Subcommittee". Currently, this group only adjudicates issues related to the CoC but it is written into the description of the committee that they may also be involved with developing and adjudicating other conduct related policies. We decided to leave the name broad to enable this expansion.